ProcessMultipliers/lowResProcessMultipliers.py

```diff
@@ -0,0 +1,174 @@
+from osgeo import osr, gdal, gdalconst
+from osgeo.gdal_array import BandReadAsArray, CopyDatasetInfo, BandWriteArray
+from netCDF4 import Dataset
+import os
+import xarray as xr
+import numpy as np
+from tqdm import tqdm
+from Utilities.config import ConfigParser
+from Utilities.files import flStartLog
+import argparse
+from os.path import join as pjoin, dirname, realpath, isdir, splitext
+import traceback
+import logging as log
+
+
+def downscale_multipliers(src_file, match_file, dst_file):
+    """
+    Downscales and clips GDAL compatable file and saves it as a GEOTIFF.
+
+    Params:
+        - src_file: filepath of the input file to be downscaled
+        - match_file: filepath of the file that the input is transformed to match
+        - dst_file: output filepath
+    """
+    # load src
+    src = gdal.Open(src_file, gdal.GA_ReadOnly)
+
+    # load match info
+    ncobj = Dataset(match_file, 'r')
+    lat = ncobj.variables['lat'][:]
+    lon = ncobj.variables['lon'][:]
+    delta = lon[1] - lon[0]
+    lon = lon - delta / 2.
+    lat = lat - delta / 2.
+
+    dx = lon[1] - lon[0]
+    dy = lat[1] - lat[0]
+    originX, originY = lon[0], lat[0]
+    epsg = 4326
+
+    wide = len(lon)
+    high = len(lat)
+
+    # Output / destination
+    srs = osr.SpatialReference()
+    srs.ImportFromEPSG(epsg)
+
+    # warp
+    drv = gdal.GetDriverByName('GTiff')
+    dst = drv.Create(dst_file, wide, high, 8, gdal.GDT_Float32)
+    dst.SetGeoTransform((originX, dx, 0, originY, 0, dy))
+    dst.SetProjection(srs.ExportToWkt())
+    dstBand = dst.GetRasterBand(1)
+    dstBand.SetNoDataValue(-9999)
+
+    gdal.ReprojectImage(src, dst, src.GetProjection(), dst.GetProjection(), gdalconst.GRA_Bilinear)
```

Comment:
Needs to have exception handling around it - including testing source file exists, destination path exists, etc.